babel/src/main/codegen/config.fmpp

```diff
@@ -1027,6 +1027,10 @@ data: {
       "DateTimeTerm()"
     ]
 
+    alternativeTypeConversionQueryMethods: [
```

Comment:
With this name, are you expecting there will be more entries in "alternativeTypeConversionQueryMethods" in the future?

Reply:
No this should be the only method for this specific syntax. Could make it clearer if I convert this to a boolean